### PR TITLE
Sssilviaa/readme

### PR DESCRIPTION
**Pull request by SSSilviaa**

### CODE REVIEW

**Comment by christianvluu** [comment]
Can you also edit "Installation" section to include the steps you followed last time?

**Comment by SSSilviaa** [comment]
> Can you also edit "Installation" section to include the steps you followed last time?

Oops. I am very sorry. If you are referring to the set up steps you so kindly helped me do last time, I barely remember all the complicated steps. But would happily edit it if we could talk it over next meeting.

**Comment by christianvluu** [comment]
> > Can you also edit "Installation" section to include the steps you followed last time?
> 
> Oops. I am very sorry. If you are referring to the set up steps you so kindly helped me do last time, I barely remember all the complicated steps. But would happily edit it if we could talk it over next meeting.

Let's talk over it next meeting. Please remind me about this.

**Comment by christianvluu** [comment]
@Jackack for visibility

File: README.md
Comment (Jackack) [inline README.md:1]: Not everyone's Robobuggy directory will be located in the same place on their machine, so the cd instructions won't work for everyone. We should just leave it to the reader to navigate to their Robobuggy folder. Also, why do we need to open file explorer?

**Comment by christianvluu** [comment]
Can you also remove the section on the 3D simulator. That is deprecated.

**Comment by christianvluu** [comment]
I think there's a way to insert those youtube videos I made into the readme directly without linking it to YouTube - that might be helpful.

**Comment by christianvluu** [comment]
@Jackack visibility.

**Comment by SSSilviaa** [comment]
> Can you also remove the section on the 3D simulator. That is deprecated.

removed now

**Comment by SSSilviaa** [comment]
> I think there's a way to insert those youtube videos I made into the readme directly without linking it to YouTube - that might be helpful.

I succeeded by half. The video can play by clicking on the image now. However, then the video fills the whole screen and cannot become small again.

**Comment by christianvluu** [comment]
Not sure if you're not seeing it but there's a few comments left I made above that are unresolved.

**Comment by SSSilviaa** [comment]
> Not sure if you're not seeing it but there's a few comments left I made above that are unresolved.

Which ones? I don’t think I see them

**Review by christianvluu — CHANGES_REQUESTED** [review]
Please See comments

File: README.md [hunk 1]
```diff
@@ -9,31 +9,45 @@ A complete re-write of the old RoboBuggy.
...
+- Type `cd` to go to the WSL's default directory.
+- Type `cd Robobuggy2`.
+- Then do `explorer.exe .` to open the file explorer to the /Robobuggy2 directory. 
+- Then do `./setup_dev.sh` in the main directory (RoboBuggy2) to launch the docker container.
+- Then you can go in the docker container using the `docker exec -it robobuggy2-main-1 bash`.
+- When you are done, type Ctrl+C and use `$exit` to exit.
+
+  ## 2D Simulation
+- To run 2D simulation on Foxglove, use `$ roslaunch buggy sim_2d.launch`.
+- Open Foxglove, choose the third one to start link.
```
[the remark this answers is not in the document]
Comment (SSSilviaa) [reply README.md]: Can you replace my screenshot of the start link with your english version? Somehow I cannot change the language. And where can I find the URL to connect?

**Comment by christianvluu** [comment]
Thanks for the edits. But for next time, please don't resolve these messages on your own - the idea is the reviewer who looks at them initially resolves them.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (christianvluu) [reply README.md]: Your screen shot is fine, URL ws://localhost:8765

[the remark this answers is not in the document]
Comment (christianvluu) [reply README.md]: Actually, this is fine. I wouldn't worry about it :)

**Review by christianvluu — APPROVED** [review]
Thank you for the changes! Good work! :) We'll get you up to speed on the next task soon.

@Jackack for visibility

**Pull request merged**